### PR TITLE
package fusesoc

### PR DESCRIPTION
**Pull request by hmaarrfk**
and a ton of other stuff i needed to get this installed

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fusesoc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - pytest >=3.3.0
+    - pyyaml
+    - simplesat >=0.8.0
+
+test:
+  imports:
+    - fusesoc
+
+about:
+  home: https://github.com/olofk/fusesoc/blob/master/LICENSE
```
Comment (chrisburr) [inline recipes/fusesoc/meta.yaml]:
```suggestion
  home: https://github.com/olofk/fusesoc/
```

File: recipes/fusesoc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  license: GPLv3
```
Comment (chrisburr) [inline recipes/fusesoc/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-3.0
```

File: recipes/zipfile2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - zipfile2
+
+about:
+  home: https://github.com/enthought/okonomiyaki
+  license: PSF2
```
Comment (chrisburr) [inline recipes/zipfile2/meta.yaml]:
```suggestion
  license: Python-2.0
```

**Comment by hmaarrfk** [comment]
Thanks @chrisburr

**Pull request merged**